core/roboconf-agent/src/test/java/net/roboconf/agent/internal/AgentMessageProcessor_BasicTest.java

```diff
@@ -76,6 +79,23 @@ public void stopAgent() {
 
 
 	@Test
+	public void testDmPingResponse() throws IllegalAccessException {
+		// Simulate a ping message from the DM.
+		MsgEcho ping = new MsgEcho( "PING:TEST" );
+		AgentMessageProcessor processor = (AgentMessageProcessor) agent.getMessagingClient().getMessageProcessor();
+		processor.processMessage( ping );
+
+		// Check the agent has sent a 'PONG' response to the DM.
+		List<Message> messages = this.client.messagesForTheDm;
+		Assert.assertTrue( messages.size() == 1 );
```

Comment:
Use Assert.assertEquals.
In case of error, the Maven/JUnit report will be more precise.